### PR TITLE
CLOUDSTACK-10287 package require open jdk 1.8

### PR DESCRIPTION
**Pull request by DaanHoogland**
## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

## Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/apache/cloudstack/blob/master/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

@blueorangutan package

### CODE REVIEW

File: packaging/centos7/cloud.spec [hunk 1]
```diff
@@ -58,7 +58,7 @@ intelligent IaaS cloud implementation.
 
 %package management
 Summary:   CloudStack management server UI
-Requires: java => 1.8.0
```
Comment (rafaelweingartner) [inline packaging/centos7/cloud.spec]: Sorry if this is a silly question, but the `requires` notation needs a full package name? Is that why you needed to change it?

Comment (DaanHoogland) [reply packaging/centos7/cloud.spec]: no, the jdk is need and not just the jre

Comment (rafaelweingartner) [reply packaging/centos7/cloud.spec]: Ah sure. 
Would it be possible to use `Requires: javac >= 1.8.0`?

Comment (DaanHoogland) [reply packaging/centos7/cloud.spec]: will look into that

Comment (rhtyd) [reply packaging/centos7/cloud.spec]: No, only jre is needed but on centos java-1.8.0-openjdk is the jre pkg and java-1.8.0-openjdk-devel has the jdk.

**Pull request merged**